packages/ui/src/views/chatmessage/ChatMessage.js

```diff
@@ -104,21 +108,30 @@ export const ChatMessage = ({ open, chatflowid, isDialog }) => {
         }, 100)
     }
 
+    const handlePromptClick = async (promptStarterInput) => {
+        setUserInput(promptStarterInput)
+        handleSubmit(undefined, promptStarterInput)
```

Comment:
@vinodkiran the reason why the first click of the prompt doesnt work is because after we update state using `setUserInput`, the updated state is not directly updated in the following function inside `handleSubmit`. Updated implementation and it works now